Github pages doesn't build anmyore

From https://github.com/metafacture/metafacture-blog/actions/runs/3196812621/jobs/5219232675 :+1: 

>Run enriikke/gatsby-gh-pages-action@v2
  with:
    access-token: ***
    deploy-branch: gh-pages
    skip-publish: false
    working-dir: .
Installing your site's dependencies using npm.
/usr/local/bin/npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: gatsby-remark-autolink-headers@5.4.0
npm ERR! Found: gatsby@3.14.6
npm ERR! node_modules/gatsby
npm ERR!   gatsby@"^3.14.6" from the root project
npm ERR!   peer gatsby@"^3.0.0-next.0" from babel-plugin-remove-graphql-queries@3.14.0
npm ERR!   node_modules/babel-plugin-remove-graphql-queries
npm ERR!     babel-plugin-remove-graphql-queries@"^3.14.0" from gatsby@3.14.6
npm ERR!     babel-plugin-remove-graphql-queries@"^3.0.0" from gatsby-plugin-image@1.0.0
npm ERR!     node_modules/gatsby-plugin-image
npm ERR!       gatsby-plugin-image@"^1.0.0" from the root project
npm ERR!     1 more (gatsby-plugin-typescript)
npm ERR!   19 more (gatsby-plugin-feed, gatsby-plugin-gatsby-cloud, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer gatsby@"^4.0.0-next" from gatsby-remark-autolink-headers@5.4.0
npm ERR! node_modules/gatsby-remark-autolink-headers
npm ERR!   gatsby-remark-autolink-headers@"^5.3.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: gatsby@4.24.2
npm ERR! node_modules/gatsby
npm ERR!   peer gatsby@"^4.0.0-next" from gatsby-remark-autolink-headers@5.4.0
npm ERR!   node_modules/gatsby-remark-autolink-headers
npm ERR!     gatsby-remark-autolink-headers@"^5.3.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -10,7 +10,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
+      - uses: actions/setup-node@v2
+        with:
+           node-version: 15
       - uses: enriikke/gatsby-gh-pages-action@v2
         with:
-          access-token: ${{ secrets.ACCESS_TOKEN }}
+          access-token: x-access-token:${{ secrets.GITHUB_TOKEN }}
           deploy-branch: gh-pages

--- a/package.json
+++ b/package.json
@@ -20,7 +20,7 @@
     "gatsby-plugin-offline": "^4.0.0",
     "gatsby-plugin-react-helmet": "^4.0.0",
     "gatsby-plugin-sharp": "^3.0.0",
-    "gatsby-remark-autolink-headers": "^5.3.0",
+    "gatsby-remark-autolink-headers": "^4.11.0",
     "gatsby-remark-copy-linked-files": "^3.0.0",
     "gatsby-remark-images": "^4.0.0",
     "gatsby-remark-prismjs": "^4.0.0",

